src/core/tsi/s2a/record_protocol/s2a_crypter.cc

```diff
@@ -148,7 +148,7 @@ static grpc_status_code derive_key(uint16_t ciphersuite, uint8_t* secret,
                                    size_t secret_size, size_t output_size,
                                    uint8_t* output, char** error_details) {
   GPR_ASSERT(error_details != nullptr);
-  static const uint8_t key_suffix[] = "\x09tls13 key\x00";
+  static uint8_t key_suffix[] = "\x09tls13 key\x00";
```

Comment:
Any reason why this is not `const` anymore?

Reply:
The compiler gave me an error saying I can't pass this `const uint8_t*` into a function expecting a `uint8_t*`. I was confused why this error only appeared after a couple of merges occurred, but had to remove the `const` for that reason.